ui/src/literature/components/BookSeriesInfoList.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+
+import InlineList, { SEPARATOR_AND } from '../../common/components/InlineList';
+import BookSeriesInfo from './BookSeriesInfo';
+
+class BookSeriesInfoList extends Component {
+  render() {
+    const { bookSeries, wrapperClassName } = this.props;
+    return (
+      <InlineList
```

Comment:
needs also `extractKey` function too probably should be combination of `title` and `volume`

Reply:
:+1: